R/join_keys-extract.R

```diff
@@ -244,7 +244,8 @@
   for (ds2 in names(norm_value)) {
     if (ds2 == i) next
 
-    keep_value <- new_x[[ds2]] %||% list()
+    keep_value <- if (is.null(x)) list() else new_x[[ds2]]
```

Comment:
Did you mean this?
```suggestion
    keep_value <- if (is.null(new_x[[ds2]])) list() else new_x[[ds2]]
```

Reply:
Thanks, please commit in case you find something